package.json

```diff
@@ -115,8 +115,8 @@
     "d3": "3.5.6",
     "d3-cloud": "1.2.1",
     "dragula": "3.7.0",
-    "elasticsearch": "13.0.1",
-    "elasticsearch-browser": "13.0.1",
+    "elasticsearch": "14.2.0",
```

Comment:
__question:__ I see we started to use `^` versions with `yarn`, can we use `^14.2.0` here as well?

Reply:
I was concerned originally since esjs changes the default api version without major version bumps, but we should be fine with yarn.lock and we should be specifying the `apiVersion` config everywhere it matters.